openapi-cli/src/main/java/io/ballerina/openapi/generators/client/FunctionBodyGenerator.java

```diff
@@ -756,7 +756,7 @@ private VariableDeclarationNode getMapForParameters(List<Parameter> parameters,
 
     private boolean checkImportDuplicate(List<ImportDeclarationNode> imports, String module) {
         for (ImportDeclarationNode importModule:imports) {
-            if (importModule.toString().equals("importballerina/" + module + ";")) {
+            if (importModule.toString().equals("import ballerina/" + module + ";")) {
```

Comment:
Shall we only assert the module name (and the org name too?), instead of doing string comparison using the whole source code of the import node?

Reply:
Fix with  d244c68